### PR TITLE
Docs update #85

### PR DESCRIPTION
**Pull request by f41gh7**
https://github.com/VictoriaMetrics/operator/issues/85

### CODE REVIEW

File: docs/backups.MD [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMSingle
+metadata:
+  name: example-vmsingle
+spec:
+  # Add fields here
+  retentionPeriod: "1"
+  vmBackup:
+    destintaion: "s3://your_bucket/folder"
```
Comment (ThomasADavis) [inline docs/backups.MD]: destintaion -> destination

File: docs/backups.MD [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+kind: VMCluster
+metadata:
+  name: example-vmcluster-persistent
+spec:
+  retentionPeriod: "4"
+  replicationFactor: 2
+  vmstorage:
+    replicaCount: 2
+    vmBackup:
+      destintaion: "s3://your_bucket/folder"
```
Comment (ThomasADavis) [inline docs/backups.MD]: same here.

File: docs/backups.MD (hunk 1 above)
Comment (f41gh7) [reply docs/backups.MD]: Thanks!

File: docs/backups.MD (hunk 2 above)
Comment (f41gh7) [reply docs/backups.MD]: There was a problem with CRD tag naming, fixed it. Thanks!

**Review by tenmozes — APPROVED** [review]

**Pull request merged**